content/pt-br/versioncontrol.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Controle de Versão
+status: Completed
+category: Tecnologia
+---
+
+## O que é
+
+O controle de código (ou controle de versão) é a prática de rastrear e gerenciar alterações de um documento. É um sistema que registra alterações de um arquivo ou conjunto de arquivos ao longo do tempo para que você possa recuperar versões específicas mais tarde.
+
+## Problema relacionado
+
+Os sistemas de controle de versão resolvem os seguintes problemas fazendo backup do documento ou do código à medida que ele muda ao longo do tempo, permitindo que vários usuários resolvam conflitos quando há alterações sobrepostas e armazenando um registro de alterações ao longo do tempo. O código do aplicativo muitas vezes pode ser complexo e crítico para os processos principais do negócio, por isso é importante rastrear quem mudou o quê, quando foi alterado e por quê. Além disso, muitos, se não a maioria dos aplicativos, são modificados por vários desenvolvedores, e muitas vezes há conflitos entre as alterações introduzidas por diferentes desenvolvedores.
```

Comment:
`resolvem os seguintes problemas fazendo backup...` can be `resolvem os seguintes problemas: backup do documento ou do código à medida que ele muda ao longo do tempo, permite que vários usuários resolvam conflitos quando há alterações sobrepostas e armazenando um registro de alterações ao longo do tempo.`